logger/consoleFormatter.go

```diff
@@ -24,14 +24,30 @@ func (cf *ConsoleFormatter) Output(line LogLineHandler) []byte {
 
 	level := LogLevel(line.GetLogLevel())
 	levelColor := getLevelColor(level)
+	timestamp := displayTime(line.GetTimestamp())
+	loggerName := formatLoggerName(line.GetLoggerName())
+	message := formatMessage(line.GetMessage())
+	args := formatArgs(levelColor, line.GetArgs()...)
 
-	return []byte(fmt.Sprintf("\033[%s%s\033[0m[%s] %s %s\n",
-		levelColor,
-		level,
-		displayTime(line.GetTimestamp()),
-		formatMessage(line.GetMessage()),
-		formatArgs(levelColor, line.GetArgs()...),
-	),
+	const formatStringWithCorrelationElements = "\033[%s%s\033[0m[%s] %s %s %s %s\n"
```

Comment:
why not taking out these consts on the package level?

Reply:
Fixed (comment above).